src/specs/src/board.spec.js

```diff
@@ -0,0 +1,168 @@
+import Canvanoid from "./../../Canvanoid.js";
+import Board from "./../../logic/Board.js";
+import Ball from "./../../logic/Ball.js";
+import Brick from "./../../logic/Brick.js";
+import State from "./../../logic/State.js";
+
+var expect = require('chai').expect;
+var should = require('chai').should();
+var assert = require('assert');
```

Comment:
@brauliodiez, @Sirikon, I have fixed this for the expect require. However, it does not seem to work for _should()_. I will keep looking into it.